target/linux/mvebu/cortexa9/base-files/etc/board.d/02_network

```diff
@@ -36,11 +34,7 @@ solidrun,clearfog-pro-a1)
 	# eth0 is standalone ethernet
 	# eth1 is switch
 	# eth2 is SFP
-	ucidef_set_interfaces_lan_wan "eth1" "eth0 eth2"
-
-	# switch port 5 is connected to eth1
-	ucidef_add_switch "switch0" \
-		"0:lan:5" "1:lan:4" "2:lan:3" "3:lan:2" "4:lan:1" "5u@eth1" "6:lan:6"
+	ucidef_set_interfaces_lan_wan "lan1 lan2 lan3 lan4 lan5 lan6" "eth0 eth2"
```

Comment:
I'm confused on this. Is this set up like the Turris Omnia? That is, 5 switched ports and one dedicated one. If so, I would set it up similar to that.

Reply:
Just read the comments. Never mind.